Hackerrank_problems/Extra Long Factorials/solution.C

```diff
@@ -0,0 +1,26 @@
+int main()
```

Comment:
As mentioned in #312, Please add comments explaining the logic to this solution of `Extra long factorials`
The other solutions are fine.

Reply:
added it.